flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -101,6 +107,7 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         }
 
         PrivateSentrySDKOnly.appStartMeasurementHybridSDKMode = true
+        PrivateSentrySDKOnly.framesTrackingMeasurementHybridSDKMode = true
```

Comment:
We should only enable these 2 flags if the performance is enabled on Flutter, Similar to RN.
https://github.com/getsentry/sentry-react-native/blob/main/ios/RNSentry.m#L88-L94